opal/mca/accelerator/base/accelerator_base_select.c

```diff
@@ -32,116 +32,108 @@
 #include "opal/mca/mca.h"
 #include "opal/mca/accelerator/base/base.h"
 #include "opal/mca/accelerator/accelerator.h"
+#include <string.h>
 
-typedef struct accelerator_opened_component_t {
+typedef struct accelerator_component_t {
   opal_list_item_t super;
   opal_accelerator_base_component_t *accelerator_component;
-  opal_accelerator_base_module_t *accelerator_module;
```

Comment:
Really confused, didn't you just add this in the last commit? Why does one PR have a change that was added and removed?

Reply:
I think this change is gone

Reply:
gone b/c of re-ordering